cirq-google/cirq_google/experimental/ops/coupler_pulse.py

```diff
@@ -117,16 +133,37 @@ def _resolve_parameters_(
             coupling_mhz=cirq.resolve_parameters(self.coupling_mhz, resolver, recursive=recursive),
             rise_time=cirq.resolve_parameters(self.rise_time, resolver, recursive=recursive),
             padding_time=cirq.resolve_parameters(self.padding_time, resolver, recursive=recursive),
+            q0_detune_mhz=cirq.resolve_parameters(
+                self.q0_detune_mhz, resolver, recursive=recursive
+            ),
+            q1_detune_mhz=cirq.resolve_parameters(
+                self.q1_detune_mhz, resolver, recursive=recursive
+            ),
         )
 
     def _value_equality_values_(self) -> Any:
-        return self.hold_time, self.coupling_mhz, self.rise_time, self.padding_time
+        return (
+            self.hold_time,
+            self.coupling_mhz,
+            self.rise_time,
+            self.padding_time,
+            self.q0_detune_mhz,
+            self.q1_detune_mhz,
+        )
 
     def _circuit_diagram_info_(self, args: cirq.CircuitDiagramInfoArgs) -> Tuple[str, ...]:
         s = f'/‾‾({self.hold_time}@{self.coupling_mhz}MHz)‾‾\\'
         return (s, s)
 
     def _json_dict_(self):
         return cirq.obj_to_dict_helper(
-            self, ['hold_time', 'coupling_mhz', 'rise_time', 'padding_time']
+            self,
+            [
+                'hold_time',
+                'coupling_mhz',
+                'rise_time',
+                'padding_time',
+                'q0_detune_mhz',
+                'q1_detune_mhz',
```

Comment:
Will this fail to deserialize old CouplerPulse objects without these values?

Reply:
I added a test that demonstrates that deserialization continues to work: `test_coupler_pulse_json_deserialization_defaults_on_missing_fields`